parse_mem.py

```diff
@@ -157,30 +96,50 @@ def create_rsa(self, output):
             univ.Integer(e1),
             univ.Integer(e2),
             univ.Integer(c),
-            )
+        )
 
-        struct = univ.Sequence()
+        s = univ.Sequence()
 
-        for i in xrange(len(seq)):
-            struct.setComponentByPosition(i, seq[i])
+        for i in range(len(seq)):
+            s.setComponentByPosition(i, seq[i])
 
-        raw = encoder.encode(struct)
+        raw = encoder.encode(s)
         data = base64.b64encode(raw)
 
         # chop data up into lines of certain width
         width = 64
-        chopped = [data[i:i + width] for i in xrange(0, len(data), width)]
+        chopped = [data[i:i + width].decode('utf-8') for i in range(0, len(data), width)]
         # assemble file content
-        content = """-----BEGIN RSA PRIVATE KEY-----
-%s
------END RSA PRIVATE KEY-----
-""" % '\n'.join(chopped)
-        output = open(output, 'w')
-        output.write(content)
-        output.close()
-
-# MAIN
-
-keystart = sshkeyparse()
-keystart.read(sys.argv[1])
-keystart.getkeys(sys.argv[2])
+        content = "-----BEGIN RSA PRIVATE KEY-----" \
+                  "\n{}\n" \
+                  "-----END RSA PRIVATE KEY-----".format('\n'.join(chopped))
+        print(content)
+        output_file_path = open(output_file_path, 'w')
+        output_file_path.write(content)
+        output_file_path.close()
+
+    @staticmethod
+    def get_dsa_key(output_file_path):
+        """
+        Create DSA SSH key file
+        """
+        print("DSA SSH Keys are not currently supported.")
+
+    @staticmethod
+    def unpack_bigint(buf: bytes):
+        """
+        Turn binary chunk into integer
+        """
+        v = 0
+
+        for c in buf:
+            v *= 256
+            v += c
+        return v
+
+
+t = SshKeyParse()
```

Comment:
L141-L145 should be moved to a `main()` method with an `if __name__ == __main__:` or similar.